packages/manager/apps/hub/src/dashboard/dashboard.controller.js

```diff
@@ -1,5 +1,22 @@
+import { DOMAINS_REPRACING, PRODUCT_TYPE_DOMAIN } from './dashboard.constants';
```

Comment:
Spell mistake: same as  https://github.com/ovh/manager/pull/7561/files#r924250726

Reply:
it's fixed